doc/release/1.16.0-notes.rst

```diff
@@ -61,6 +61,12 @@ Now, if the `NPY_DISTUTILS_APPEND_FLAGS` environment variable is set to 1, the
 behavior will be appending.  This applied to: `LDFLAGS`, `F77FLAGS`,
 `F90FLAGS`, `FREEFLAGS`, `FOPT`, `FDEBUG`, and `FFLAGS`.  See gh-11525 for more
 details.
+``np.clip`` and the ``clip`` method check for memory overlap
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ``out`` argument to these functions is not always tested for memory overlap
```

Comment:
"is not" -> "is now" or "was not".  I would omit the "Compare..." sentence; I don't think that helps the average reader much. If you keep it: "Compare also *with* ..."